src/redisai.c

```diff
@@ -891,7 +892,8 @@ int RedisAI_DagRun_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv, i
     if (RedisModule_IsKeysPositionRequest(ctx)) {
         return RedisAI_DagRun_IsKeysPositionRequest_ReportKeys(ctx, argv, argc);
     }
-    return ProcessRunCommand(ctx, argv, argc, CMD_DAGRUN, REDISAI_DAG_WRITE_MODE);
+    bool ro_only = false;
+    return RedisAI_ExecuteCommand(ctx, argv, argc, CMD_DAGRUN, ro_only);
```

Comment:
you don't have to declare the variable - you can just send it in the function
leave or modify at your discretion.